htdocs/knowledgemanagement/knowledgerecord_list.php

```diff
@@ -117,6 +118,15 @@
 			'position'=>$val['position'],
 			'help'=> isset($val['help']) ? $val['help'] : ''
 		);
+	} elseif ($key = 'lang') {
```

Comment:
Include this test into the 
if (!empty($val['visible'])) {
and do the content of the if (!empty...) into the else.